### ISSUE TITLE
MaryTTS Russian voice

### ISSUE THREAD

**Issue by 5LICK** [openhab/openhab#3231]
Hi, Please add russian voice for MaryTTS, i checked last release MaryTTS and i found russian language and voice for it.

**Comment by nurgaleevarman** [comment]
You can use googletts addon to russian voice.
Look https://github.com/openhab/openhab/issues/3301 for correct config file

**Comment by 5LICK** [comment]
Yes, but i want have a local smart home system. That would be good

**Comment by nurgaleevarman** [comment]
I tried the following:
1) Download MaryTTS Server. Install Russian voice via marytts-5.1.2\bin\marytts-component-installer.bat
2) Add add-on GoogleTTS
3)In Openhab config fie move to section GoogleTTS
set next
<code>
googletts:language=RU
</code>
<code>
googletts:sentenceDelimiters=+
</code>
<code>
googletts:translateUrl=http://localhost:59125/process?LOCALE=%s&INPUT_TEXT=%s&INPUT_TYPE=TEXT&OUTPUT_TYPE=AUDIO&AUDIO=WAVE_FILE
</code>

Openhab log
<code>
2015-10-17 15:46:21.123 [INFO ] [runtime.busevents             ] - MySwitch received command ON
</code>
<code>
2015-10-17 15:46:21.138 [INFO ] [.i.m.i.tts.TTSServiceGoogleTTS] - Executing GoogleTTS for text 'Привет Мир' in language RU
</code>

MaryTTS Server log show what request was correct and generate WAVE_FILE.
There was no sound and error in Openhab.  =) I think what audio format not equal with Google TTS.

MaryTTS Server Conversion to MP3 not supported

**Comment by 5LICK** [comment]
Thanks for this `nurgaleevarman`
Google TTS require internet connection, but i need local tts for my home (if i have problems with my internet)
